grimoire/placement/nooks/features/flora.py

```diff
@@ -0,0 +1,33 @@
+from logging import error
+from typing import Iterable
+
+from gdpc import Editor
+from gdpc.block import Block
+from gdpc.vector_tools import Box, Rect, ivec3
+
+from grimoire.core.maps import Map
+
+DEFAULT_TREE_TYPE = "minecraft:oak"
+
+
+def place_tree(
```

Comment:
I'll definitely look toward integrating that once we start development again. I think for now I'll just mark that as a To-Do comment, as I'd like to avoid non-fix changes to this branch.